src/main/java/seedu/address/logic/parser/CopyCommandParser.java

```diff
@@ -17,16 +17,15 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public CopyCommand parse(String args) throws ParseException {
-        try {
-            String[] keywords = args.split(" ");
-            Index activityIndex = ParserUtil.parseIndex(keywords[1]);
-            Index travelPlanIndex = ParserUtil.parseIndex(keywords[2]);
-
-            return new CopyCommand(activityIndex, travelPlanIndex);
-        } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, CopyCommand.MESSAGE_USAGE), pe);
+        String[] keywords = args.split(" ");
+        if (keywords.length != CopyCommand.COMMAND_TOKENS) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, CopyCommand.MESSAGE_USAGE));
         }
+
+        Index activityIndex = ParserUtil.parseIndex(keywords[1]);
+        Index travelPlanIndex = ParserUtil.parseIndex(keywords[2]);
```

Comment:
Can add constants for the magic numbers here? i.e. `ACTIVITY_INDEX_POSITION = 1 `and `TRAVELPLAN_INDEX_POSITION = 2`. Same for MoveCommandParser. Perhaps can put this in ParserUtil.

Reply:
Done!